src/main/java/org/fcrepo/audit/InternalAuditor.java

```diff
@@ -231,7 +226,18 @@ public void createAuditNode(final FedoraEvent event) throws IOException {
     }
 
     @VisibleForTesting
+    protected String getEventPath(final String eventID) {
+        if (!eventID.startsWith("urn:uuid:")) {
+            throw new IllegalArgumentException("Event ID must be a 'urn:uuid:'" + eventID);
+        }
+        return pathMinter.get(eventID.substring("urn:uuid:".length()));
```

Comment:
I would also note that the `HierarchicalIdentifierSupplier` extends `UniqueValueSupplier`, and by adding this method, it seems to break the implicit contract of supplying a _unique_ value.